Client/src/Components/TabPanels/Account/PasswordPanel.jsx

```diff
@@ -116,14 +116,6 @@ const PasswordPanel = () => {
           <Box flex={0.9}>
             <Typography component="h1">Current password</Typography>
           </Box>
-          <Field
-            type="text"
-            id="hidden-username"
-            name="username"
-            autoComplete="username"
-            hidden={true}
-            value=""
-          />
```

Comment:
This hidden text field is here for accessibility, this should be left in

Reply:
If the hidden text field is for autocomplete accessibility only, can we move it outside the <Stack component="form">?

Reply:
Is it causing the some issue where it is now? It shouldn't affect anything where it is I don't think 🤔

Reply:
It acquires the space in page which causes the current password input field inconsistent(which is the issue #907).

https://private-user-images.githubusercontent.com/167266851/373501067-aa4c9ae8-88fb-424e-bcb7-108d115a7778.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg2NDk3NzUsIm5iZiI6MTcyODY0OTQ3NSwicGF0aCI6Ii8xNjcyNjY4NTEvMzczNTAxMDY3LWFhNGM5YWU4LTg4ZmItNDI0ZS1iY2I3LTEwOGQxMTVhNzc3OC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAxMVQxMjI0MzVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jMDFiYTY3ZDdiNTdiNmQwYTZmMWE5ODJhMWZiZjgzNjI5OWFjOTVhMzZiNzMxZTA1OTYwMDI2YWM4NmZiOTVkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.xSiQlO2yceCcBy6nMmMyL4E8xDckMHAQETOk-BSvwhg

Reply:
So if the hidden field is used only for autocomplete accessibility, I think it will be fine if we don't send that field to backend when the form is submitted.

Reply:
> It acquires the space in page which causes the current password input field inconsistent(which is the issue #907).
> 
> https://private-user-images.githubusercontent.com/167266851/373501067-aa4c9ae8-88fb-424e-bcb7-108d115a7778.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg2NDk3NzUsIm5iZiI6MTcyODY0OTQ3NSwicGF0aCI6Ii8xNjcyNjY4NTEvMzczNTAxMDY3LWFhNGM5YWU4LTg4ZmItNDI0ZS1iY2I3LTEwOGQxMTVhNzc3OC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDExJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAxMVQxMjI0MzVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jMDFiYTY3ZDdiNTdiNmQwYTZmMWE5ODJhMWZiZjgzNjI5OWFjOTVhMzZiNzMxZTA1OTYwMDI2YWM4NmZiOTVkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.xSiQlO2yceCcBy6nMmMyL4E8xDckMHAQETOk-BSvwhg

Ah understood.

I think it would be best to keep it where it is so it is clear what it is for.

How about `display: none` instead of visibility hidden?  I'm not sure if that still resolves the accessibility issue or not. 

If setting display to none doesn't work, how about setting the line height of the hidden element to 0?

Reply:
Hello @ajhollid, I had some problem with this forked repo, so I have forked a new one and created a new PR #940. Please review and let me know if any changed